### DISCUSSION TITLE
"n_issues": is "-1" absence of issues?

### DISCUSSION THREAD

**Discussion by Better-L** [ep142/FoodMicrobionet#4] [Q&A] [answered]
Good morning,
from the "ShinyFMBNManual" I understand that the field "n_issues" is set to "-1" for importing reason- but in this case, is the actual number of issue absent just like when it's "0"?
I find this option really useful- a further useful information may be the detailed criteria used to define the issue: i.e. which number (percentage?) of "[..] chimeras/bimeras, high loss during quality filtering, low number of features, low number of sequences after processing".
Thank you.
All the best,
Luca

**Comment by ep142** [comment, marked as answer]
Dear Luca,
the data for some of the studies in FoodMicrobionet were contributed as OTU tables + metadata. Those are the studies whose samples have -1 in the n_issues field. I am reprocessing the raw sequences for some of this studies and thy will be available with next version. I know it would be nice to have a detail on issues but this was frankly too much work. Anyway, I keep track of this in a separate file and, eventually, if more people request this feature, I will merge these data in the sample table. Please note that an extended version of the database, which includes a folder containing lists with phyloseq objects with the raw sequences and further metadata is available on request (in the framework of scientific collaborations)
